src/frontend/src/pages/TeamsPage/TeamMembersView.tsx

```diff
@@ -0,0 +1,123 @@
+import PageBlock from '../../layouts/PageBlock';
+import { Autocomplete, Button, Grid, IconButton, TextField, useTheme } from '@mui/material';
+import { fullNamePipe } from '../../utils/pipes';
+import { Team } from 'shared';
+import { Edit } from '@mui/icons-material';
+import { useState } from 'react';
+import { useAuth } from '../../hooks/auth.hooks';
+import { useAllUsers } from '../../hooks/users.hooks';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import ErrorPage from '../ErrorPage';
+import { User } from 'shared';
+import { useSetTeamMembers } from '../../hooks/teams.hooks';
+
+interface TeamMembersViewProps {
+  team: Team;
+}
+
+const TeamMembersView: React.FC<TeamMembersViewProps> = ({ team }) => {
+  const auth = useAuth();
+  const theme = useTheme();
+  const [isEditingMembers, setIsEditingMembers] = useState(false);
+  const [members, setMembers] = useState(team.members);
+
+  const { isLoading, isError, error, data } = useAllUsers();
+  const {
+    isLoading: memberIsLoading,
```

Comment:
all of these should be `setTeamMembersIsLoading`

long variable names are good